webpack.config.js

```diff
@@ -31,6 +33,15 @@ const entry = appsFiles.reduce((obj, [filePath, name]) => {
 module.exports = {
     entry,
     resolve: {
+        alias: {
+            /**
+             * with the alias we can mimic the usage of the library like it would be in real application
+             * `import {ParticleSystem} from 'XXX`
+             *
+             * TODO: define the name of the library here
+             */
+            "@": path.resolve(__dirname, "./src/"),
```

Comment:
We should come up with a library name in this pull request, even if it is not necessarily final.

This @/ is confusing, but with a library name I think it will be look intuitive.

Reply:
Yes, something I wanted to discuss ;) 

Simply because at the end of the day, the end-user would use the proper import rather than '@'

So, the examples treat core as node module